src/containers/Repo/Grader/GraderPage.js

```diff
@@ -117,6 +131,7 @@ export default class GraderPage extends Component {
         graderId,
         repoId
       });
+      this.setState({ showModal: true });
```

Comment:
I don't want the modal to show up when you click the buttons. I want the modal to show up after we get a response back from the server. You need to do the check in componentsWillReceiveProps and check if submission is good.